Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift

```diff
@@ -1777,7 +1777,7 @@ fileprivate final class TokenStreamCreator: SyntaxVisitor {
 
     let wrapsBeforeOperator = !isAssigningOperator(binOp)
 
-    if shouldRequireWhitespace(around: binOp) {
+    if shouldRequireWhitespace(around: binOp, configuration: config) {
```

Comment:
Since `config` is a member of `TokenStreamCreator`, you don't need to add it as an argument to this function; it would be cleaner to just refer to the member directly inside it.